lib/formtools/renderers-cell.js

```diff
@@ -156,7 +156,17 @@ var documentArrayRenderer = module.exports.documentArray = function documentArra
 
     });
 
-}
+};
+
+var textRenderer = module.exports.text = function textRenderer(val, record, fieldName, model, callback) {
+
+    if (typeof val === 'undefined') {
```

Comment:
You can remove this `if` statement, as `L167` would pass `undefined` as the `val` anyways.